src/web/services/userAccount.ts

```diff
@@ -46,9 +36,9 @@ export async function GetLoggedInUserAccount(): Promise<UserWithIsApprover | nul
   }
 }
 
-export async function ResendInvite(id: number): Promise<void> {
+export async function ResendInvite(id: number, participantId: number): Promise<void> {
```

Comment:
This function doesn't strictly need `participantId` for it to work, but I figured it still be useful for audit trail purposes.